x-pack/plugins/cases/server/saved_object_types/migrations/index.ts

```diff
@@ -148,49 +148,43 @@ export const createCommentsMigrations = (
   ) as MigrateFunctionsObject;
 
   const commentsMigrations = {
-    '7.11.0': flow(
```

Comment:
I don't think we need the `flow` calls. We don't have them in any other place within cases.